lightstep/instrument.py

```diff
@@ -177,12 +178,27 @@ def _flush_worker(self, connection):
 
     def _construct_report_request(self):
         """Construct a report request."""
+        report = None
         with self._mutex:
             report = ttypes.ReportRequest(self._runtime, self._span_records,
                                           self._log_records)
             self._span_records = []
             self._log_records = []
-            return report
+
+        for log in report.log_records:
+            if log.stable_name is not None:
+                log.stable_name = str(log.stable_name)
+            if log.payload_json is not None:
+                try:
+                    log.payload_json = \
+                                       jsonpickle.encode(log.payload_json,
+                                                         unpicklable=False,
+                                                         make_refs=False,
+                                                         max_depth=constants.JSON_MAX_DEPTH)
+                except:
+                    log.payload_json = jsonpickle.encode(constants.JSON_FAIL)
```

Comment:
(https://github.com/resonancelabs/reslabs/blob/master/doc/specs/lightstep_binding_library.md#spec has something about this under the **Reporting** sub-section)